op-conductor-ops/op-conductor-ops.py

```diff
@@ -97,10 +97,14 @@ def status(network: str):
                     print_error(
                         f": {sequencer.sequencer_id} does not have the correct voting status.")
                     display_correction = True
+                del membership[sequencer.sequencer_id]
```

Comment:
Slightly ambiguous naming imo. Something like `untracked_members` would be more relevant imo.

But definitely not blocking this PR. So feel free to address or ignore this comment :)